src/app/containers/app/app.component.ts

```diff
@@ -81,6 +88,16 @@ export class AppComponent implements OnInit, OnDestroy {
     }
   }
 
+  // EUI-4069 - Get roles to display on allocate role journey one time on login
+  public setRoleAllocations(): void {
```

Comment:
1. It doesn't but was requested by Kuda. If you think it would be more appropriate to split these calls then maybe we should have a call and include Kuda?
2. When discussing with Andy he mentioned we should not be making multiple calls for this as the roles are unlikely to change. take(1) ensures that we only get the initial returned result.
Do you still want me to change it?

Reply:
yes let's have a call tomorrow.